test/e2e/density.go

```diff
@@ -80,10 +80,41 @@ var _ = Describe("Density", func() {
 	var additionalPodsPrefix string
 	var ns string
 	var uuid string
-	framework := Framework{BaseName: "density", NamespaceDeletionTimeout: time.Hour}
+
+	AfterEach(func() {
+		// We can't call it explicitly at the end, because it will not be called
```

Comment:
this comment is obsolete, but it might be worth having a note why this AfterEach comes before everything else